babel/numbers.py

```diff
@@ -317,70 +317,163 @@ def _is_active(start, end):
     return result
 
 
-def get_decimal_symbol(locale: Locale | str | None = LC_NUMERIC) -> str:
+def _get_numbering_system(locale: Locale, numbering_system: Literal["default"] | str = "latn") -> str:
+    if numbering_system == "default":
+        return locale.default_numbering_system
+    else:
+        return numbering_system
+
+
+def _get_number_symbols(
+    locale:  Locale | str | None = LC_NUMERIC,
```

Comment:
I don't think this internal function should have a default here.
```suggestion
    locale: Locale | str,
```